spec/helpers/stylesheet.js

```diff
@@ -14,12 +16,23 @@ describe('stylesheet helper', () => {
     const runTestCases = testRunner({siteSettings});
 
     it('should render a link tag with the cdn ulr and stencil-stylesheet data tag', done => {
-        runTestCases([
+        const renderer = buildRenderer(siteSettings);
+        const runner = testRunner({renderer});
```

Comment:
I think it's possible to do something like this (before all test cases)
```
describe('....', () => {
  let renderer, runner;
  beforeEach(() => {
     renderer = buildRenderer(siteSettings);
     runner = testRunner({renderer});
  });
  
  it('test 1', () => {
     runner([], done);
  });
   it('test 2', () => {
     runner([], done);
  });
}
```
So before each test case new instance of `renderer` and `runner` will be created and the state won't be preserved